client/JSDebugger/mlRuntime.ts

```diff
@@ -129,6 +130,7 @@ export class MLRuntime extends EventEmitter {
             new MlClientParameters({
                 host: this._hostName,
                 port: this._dbgPort,
+                restBasePath: this.manageBasePath,
```

Comment:
Is this correct? I see how `MlClientParameters` has both `restBasePath` and `manageBasePath` properties - this seems odd because I would expect these two values to be different.

Reply:
It's definitely convoluted, but it is correct.

MlClientParameters is capable of holding all of the MLXPRS settings individually.
However, when it is used to build a ClientContext object, only the "port" and "restBasePath" are used. The other ports/paths are ignored. So, I need to set "port" and "restBasePath" to the values I need for this specific client.
In this case, I need a client using the "manage" parameters.

One refactor that I've been considering - In the constructor for ClientContext, add a second parameter specifying which port/path pair to use. That would at least clean up this wierdness.

Reply:
Got it - I think the confusion for me is I assume that `MlClientParameters` contains the params for one client object. The "mlxprs config" would have multiple ports and paths - one pair for rest/test/manage/admin - but when a `ClientParams` object is created, it should have `port` and `basePath` properties, and the caller chooses which pair of values to pass in. 

I don't know how feasible it would be to refactor `MlClientParameters` to be like that, but it would avoid confusion like `restBasePath=manageBasePath`.